src/css/blog-post.module.scss

```diff
@@ -0,0 +1,204 @@
+@import "/src/css/colors.scss";
+@import "/src/css/text-styles.scss";
+
+// Header Blog Elements
+
+.headerContainer {
+  padding: 5rem 20rem 10rem;
+}
+
+.titleContainer {
+  margin: 3rem 0;
+}
+
+.authorContainer {
+  width: 100%;
+  max-height: 5.5rem;
+}
+
+.authorInformation {
+  display: flex;
+  flex-direction: row;
+}
+
+.authorImage {
+  border-radius: 10px;
+  margin-right: 2rem;
+  width: 5.5rem;
+  height: 100%;
+}
+
+.authorName {
+  text-decoration: none;
+}
+
+.postDate {
+  display: flex;
+  justify-content: end;
+  position: relative;
+  bottom: 1.7rem;
+}
+
+// Blog body
+
+.bodyPostContainer {
+  padding: 5rem 20rem 10rem;
+  text-align: justify;
+
+  h1 {
+    @extend .text__heading__one__black;
```

Comment:
If I'm not mistaken you already import these classes in the `jsx` file, right?

Reply:
No, this is the section for the body of the blog, I used the classes for the banner section.